chef/cookbooks/provisioner/templates/default/autoyast-upgrade.xml.erb

```diff
@@ -108,6 +108,20 @@
         </source>
       </script>
 <% end -%>
+      <script>
+        <chrooted config:type="boolean">true</chrooted>
+        <debug config:type="boolean">true</debug>
+        <filename>fix_alternatives</filename>
+        <source>
+          <![CDATA[
+          # python-related alternatives are not updated, even though the binary
+          # names changed (2.6 to 2.7); find all broken links and fix/update them
+          for i in $(find /etc/alternatives/ -type l -! -exec test -e {} \; -print | xargs basename -a) ; do
```

Comment:
It's generally safer to use `-print0`:

~~~
find /etc/alternatives/ -type l -! -exec test -e {} \; -print0 | xargs --null basename -a
~~~